### DISCUSSION TITLE
osgEarth & ImGui : A Nice Couple

### DISCUSSION THREAD

**Discussion by nebsar** [pelicanmapping/osgearth#2021] [Support Q&A]
I would like to share my very draft (baby steps) work's screenshot with you. I used osgEarth and ImGui here. They really fit into each other. This interface gets real-time UDP data from XPlane11 simulation game. Lots of things to be done, especially for the moving map side. Let's see....

![image](https://user-images.githubusercontent.com/5169314/159190494-623027d7-5ea1-4744-b999-cf5316268553.png)

**Comment by jasonbeverage** [comment]
That looks awesome!   Great job.

**Reply by nebsar** [reply]
Thanks Jason.

**Comment by gwaldron** [comment]
Very nice!!!

**Reply by nebsar** [reply]
Thanks Glenn.

**Comment by keince** [comment]
Looks great.

**Reply by nebsar** [reply]
Thanks keince.

**Comment by nebsar** [comment]
Screenshot of the current version:

![image](https://user-images.githubusercontent.com/5169314/161365978-6d706353-387b-4c97-9ae8-460aaf8304ce.png)

And a short video showing the page selection and swapping capabilities.

https://www.youtube.com/watch?v=9Df3zYNgbn0
